cmd/relayer/main.go

```diff
@@ -0,0 +1,388 @@
+// Package main is the entrypoint for the swap-specific Ethereum transaction relayer.
+// It's purpose is to allow swaps users (ETH-takers in particular) to submit calls to the
```

Comment:
`Its`. Ironically, "its" is only possessive when it doesn't have the apostrophe. :)

Reply:
updated